todo_app/app.py

```diff
@@ -1,30 +1,190 @@
-from flask import Flask
-from flask import request
+from flask import request,redirect,url_for,Response,Flask,render_template
 from todo_app.flask_config import Config
-from flask import render_template
 from todo_app.data import session_items
+import requests
+import os
+import json
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-@app.route('/todo')
-def index():
-    items = session_items.get_items()
-    return render_template('Index.html', items=items)
+def get_trello_API_credentials():
+    f = open("todo_app/Trello_API_Keys.txt", "r").read().split("\n")
+    return {'key': f[0], 'token': f[1]}
```

Comment:
The alternative approach (which will make tests easier for the next project exercise) is to use environment variables.

There is already a .env file which will automatically load env vars when you run `flask run`. So you can add a TRELLO_KEY and TRELLO_TOKEN there and read them out in your Python code via `os.getenv('TRELLO_KEY')` etc

Either way, you need to document things like this in your README!